### DISCUSSION TITLE
AWS DynamodDB V2 Support

### DISCUSSION THREAD

**Discussion by seanmurrayunity** [guregu/dynamo#177] [Q&A]
Hi! Will you be adding support for DynamoDB V2?

**Comment by guregu** [comment]
Do you mean AWS SDK v2? I'd like to support it in the next major version with breaking changes.
https://github.com/guregu/dynamo/issues/113#issuecomment-823765631
I'm not sure when that will be though, but maybe this year.

**Reply by seanmurrayunity** [reply]
Yes, AWS SDK v2. Sounds good, i'll keep an eye on this! Thank you so much for your quick response.

**Reply by conor-tread** [reply]
> Do you mean AWS SDK v2? I'd like to support it in the next major version with breaking changes. [#113 (comment)](https://github.com/guregu/dynamo/issues/113#issuecomment-823765631) I'm not sure when that will be though, but maybe this year.

Does this support v2 now?
